### ISSUE TITLE
Output nested Java classes to support single-file output

### ISSUE THREAD

**Issue by dvdsgl** [quicktype/quicktype#224]
Our multi-file output support is janky. Perhaps we could simplify the Java case by nesting our types in an outer class?

**Comment by schani** [comment]
This works only where there is a single top-level class.

**Comment by dvdsgl** [comment]
I agree about fixing the workflow, I just thought this could be an interesting shortcut for copy-pasting from the web to your IDE.

**Comment by dvdsgl** [comment]
I discovered that you can also import nested classes, so the 'really ugly' method isn't so ugly to consume.

**Comment by schani** [comment]
So maybe this should be an option that's on by default on the web?

**Comment by schani** [comment]
It's kinda nice that the outer class can then also hold the convenience methods, at least in the case of a single top-level.

**Comment by schani** [comment]
We do not take feature requests anymore. Please consider making a pull request instead. We encourage the use of coding agents, but please use at least an Opus-class model.
